program/Cargo.toml

```diff
@@ -9,7 +9,7 @@ repository = { workspace = true }
 homepage = { workspace = true }
 license = { workspace = true }
 edition = { workspace = true }
-rust-version = "1.79.0"                          # solana platform-tools rust version
+rust-version = "1.84.1"                          # solana platform-tools rust version
```

Comment:
Can we just bump this to 1.81.0 to match the requirement from program-entrypoint?